styles.css

```diff
@@ -310,3 +310,64 @@ nav {
   cursor: pointer;
   text-align: center;
 }
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.contact p {
```

Comment:
- [ ] kindly fix the text size and the icons
Your Design
![Capturetef](https://user-images.githubusercontent.com/81444338/171232285-ab224058-606b-4ffd-86ce-bc25fd6720ab.PNG)
- Figma Design
![Capturei](https://user-images.githubusercontent.com/81444338/171232978-f3f87f03-bc84-4f34-b6f3-7ce8cb8ccab8.PNG)